spec/requests/api/v0/user_collections_spec.rb

```diff
@@ -13,9 +13,9 @@
       get '/api/user_collections', params: { user_id: @user.id }
 
       expect(response).to be_successful
-
+      
       collections = JSON.parse(response.body)['collections']
-      expect(collections.count).to eq(3)
+      expect(collections.count).to eq(4)
```

Comment:
This should continue to work with the updated controller and SaveButton code. Although three articles are seeded for this test, the api creates a default collection if not already generated.